Lesson33/Ezgi/lesson33-app/src/Exercise10/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+
+export const Exercise10 = () => {
+   
+    const [clickCount, setClickCount] = useState(0);
+
+    function doubleClickHandler () {
+        setClickCount(clickCount + 1);
+
+        if(clickCount === 2) {
+            alert('Button was double-clicked!');
+            setClickCount(0);
+        }
+    }
+    return (
+        <button onClick={doubleClickHandler}>Double click me</button>
```

Comment:
Here you can use 'onDoubleClick' event listener instead

Reply:
I hadn't thought of that. I've changed it to onDoubleClick